widget/embedded/src/components/Quote/Quote.tsx

```diff
@@ -235,24 +272,26 @@ export function Quote(props: QuoteProps) {
                 size="small"
                 variant="body"
                 style={{ letterSpacing: 0.4 }}>
-                {`${input.value} ${steps[0].from.token.displayName} = ${
-                  output.value
-                } ${steps[steps.length - 1].to.token.displayName}`}
+                {`${roundedInput} ${
+                  steps[0].from.token.displayName
+                } = ${roundedOutput} ${
+                  steps[steps.length - 1].to.token.displayName
+                }`}
               </Typography>
               <Typography
                 color="$neutral600"
                 ml={2}
                 size="xsmall"
                 variant="body">
-                {`($${output.usdValue})`}
+                {`($${roundedOutputUsdValue})`}
               </Typography>
             </div>
           )}
           {type === 'list-item' && (
             <TokenAmount
               type="output"
               direction="vertical"
-              price={{ value: output.value, usdValue: output.usdValue }}
+              price={{ value: roundedOutput, usdValue: roundedOutputUsdValue }}
               token={{
```

Comment:
Maybe not related to this PR but i think output value for loading state (when user prompts for route) is wrong. It should have another state rather than 0 or 0$.

![Recording 2023-11-18 at 15 11 36](https://github.com/rango-exchange/rango-client/assets/99180934/94df9c71-f8d3-4a2a-94eb-1d07a25f0cce)